awx/api/filters.py

```diff
@@ -48,7 +48,7 @@ class TypeFilterBackend(BaseFilterBackend):
     def filter_queryset(self, request, queryset, view):
         try:
             types = None
-            for key, value in request.query_params.items():
+            for key, value in list(request.query_params.items()):
```

Comment:
I don't think we need to cast it to a list:

```
In [5]: d['a'] = 1                                                                                                 

In [6]: for k, v in d.items(): 
   ...:     print(k, v) 
   ...:                                                                                                            
a 1
```

Reply:
Agreed, looks like I got over-aggressive with some `sed` here.